Editor/MetadataWindow.cpp

```diff
@@ -172,7 +172,7 @@ void MetadataWindow::RefreshEntries()
 		Entry& entry = entries.emplace_back();
 		entry.name.Create("");
 		entry.name.SetText(name);
-		entry.name.OnInputAccepted(forEachSelectedWithRefresh(name, [&] (auto metadata, auto args) {
+		entry.name.OnInputAccepted(forEachSelectedWithRefresh(name, [&name] (auto metadata, auto args) {
```

Comment:
I see that `name` is a `std::string&` which is owned by a `std::vector<std::string>` in `MetadataComponent`. Do we know that this vector is not inserted to after capturing the reference here?

Reply:
I'm pretty sure the only way `metadata->bool_values.names` gets changed is by user action, and `RefreshEntries` only gets called after a user action; so unless the user is The Flash, it can be considered constant during the execution of this method.